projects/plugins/jetpack/_inc/client/main.jsx

```diff
@@ -854,10 +857,15 @@ class Main extends React.Component {
 					<AdminNotices />
 					<JetpackNotices />
 					{ this.shouldConnectUser() && this.connectUser() }
+					{ /*
+					This component was removed as of react-router-dom v6: https://github.com/remix-run/react-router/issues/8139
+					It could probably be brought back with `unstable_usePrompt`, but that would require jumping through hoops with
+					createHashRouter(), etc., and is already not reliable cross-browser anyway.
```

Comment:
More important than the need to switch from `<HashRouter>` to `createHashRouter()` is that it doesn't even work then anyway.
```suggestion
					It could probably be brought back with `unstable_usePrompt`, but that is broken with the hash router and normal links,
					and is already not reliable cross-browser anyway.
```